src/Platforms/PostgreSQLPlatform.php

```diff
@@ -194,17 +203,35 @@ public function supportsPartialIndexes()
 
     /**
      * {@inheritdoc}
+     *
+     * @deprecated
      */
     public function usesSequenceEmulatedIdentityColumns()
     {
+        Deprecation::trigger(
+            'doctrine/dbal',
+            'https://github.com/doctrine/dbal/pull/5513',
+            '%s is deprecated.',
+            __METHOD__
+        );
+
         return true;
     }
 
     /**
      * {@inheritdoc}
+     *
+     * @deprecated
      */
     public function getIdentitySequenceName($tableName, $columnName)
     {
+        Deprecation::triggerIfCalledFromOutside(
```

Comment:
What is the replacement for https://github.com/atk4/data/blob/d09466d79908ea8ae9c07d7dea0611cbfc2024b3/src/Persistence/Sql.php#L680 ?

Of course, it can be hardcoded, but if the name will change in DBAL, it will make the data framework not working...